GitHubUserSearch/GitHubUserSearch/Model/Model.swift

```diff
@@ -111,8 +114,7 @@ class Model {
                      self.cache.setObject(aData as AnyObject, forKey: "\(userId)" as AnyObject)
```

Comment:
Also you can make name of this method shorter like
` func getAvatar(for user: UserData, completion:  imageDownloadBlock?)`